openfecwebapp/templates/macros/filters/date.html

```diff
@@ -47,11 +47,58 @@
   <div class="filter js-filter" id="{{ name }}" data-filter="date" data-name="{{ name }}" data-validate="true">
     <fieldset>
     <legend class="label">{{ label }}</legend>
-    <div>
-      <label for="min_{{ name }}">From</label>
-      <input type="text" id="min_{{ name }}" name="min_{{ name }}" class="js-min-date" data-prefix="Beginning" data-inputmask="'alias': 'mm-dd-yyyy'">
-      <label for="max_{{ name }}">To</label>
-      <input type="text" id="max_{{ name }}" name="max_{{ name }}" class="js-max-date" data-prefix="Ending" data-inputmask="'alias': 'mm-dd-yyyy'">
+    <div class="range range--date">
+      <div class="range__input range__input--min" data-filter="range">
+        <label for="min_{{ name }}">Beginning</label>
+        <input type="text" id="min_{{ name }}" name="min_{{ name }}" data-range="min" class="js-min-date" data-prefix="Beginning" data-inputmask="'alias': 'mm/dd/yyyy'">
+      </div>
+      <div class="range__hyphen">-</div>
+      <div class="range__input range__input--max" data-filter="range">
+        <label for="max_{{ name }}">Ending</label>
+        <input type="text" id="max_{{ name }}" name="max_{{ name }}" data-range="max" class="js-max-date" data-prefix="Ending" data-inputmask="'alias': 'mm/dd/yyyy'">
+      </div>
+      <button class="button--go button--standard" type="button">
+        <span class="u-visually-hidden">Search</span>
+      </button>
+    </div>
+    <div class="date-range-grid">
+      <div class="date-range-first">
```

Comment:
Maybe `.date-range-first` and `.date-range-second` should just be `.date-range` and select for them in JS using either data attributes or `nth-of-type`?